src/app/systelab-components/select/gender-combobox.component.ts

```diff
@@ -55,4 +60,17 @@ export class GenderSelect extends AbstractComboBox<Element> implements AfterView
 	getIdField(): string {
 		return 'id';
 	}
+
+
+	deleteValueSelected(event: MouseEvent): void {
+		event.stopPropagation();
+		if (this.showAll) {
+			this._id = 'A';
```

Comment:
As you have and id, then we are going to have the close icon always. as the cross is visible when "withDeleteOption && id"